easybuild/easyblocks/c/cuda.py

```diff
@@ -129,17 +133,18 @@ def sanity_check_step(self):
             chk_libdir += ["lib"]
 
         custom_paths = {
-            'files': ["bin/%s" % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] +
-                     ["%s/lib%s.%s" % (x, y, shlib_ext) for x in chk_libdir for y in ["cublas", "cudart", "cufft",
-                                                                                      "curand", "cusparse"]],
+            'files': [os.path.join("bin", "%s") % x for x in ["fatbinary", "nvcc", "nvlink", "ptxas"]] +
+                     [os.path.join("%s", "lib%s.%s") % (x, y, shlib_ext) for x in chk_libdir for y in ["cublas",
```

Comment:
A variant of writing this is:
[os.path.join("bin", x) for x in [...]]
[os.path.join(x, "lib%s.%s") % (y, shlib_ext) for x in ...

At least the first one is slightly easier to read

Reply:
probably easier to lift the `["cublas", ...]` list into a variable:

```python
culibs = ["cublas", "cudart", "cufft", "curand", "cusparse"]
...
   [os.path.join("%s", "lib%s.%s") % (d, l, shlib_ext) for d in chk_libdir for l in culibs]